CHANGELOG.md

```diff
@@ -8,9 +8,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [develop]
 
 ### General
+- Transition from GitLab to GitHub
+- ExaGO now depends on HiOp 1.0.0. Next release will require HiOp 1.0.1+
```

Comment:
Looks like all platforms are built with hiop1.0.0. Is this dependency in the spack package now?

Reply:
The dep will have to be update in the release - we can do that quickly now, or do it together with the release.

An issue to keep us on track would be reasonable

Reply:
See #20

Reply:
you mean #45

Reply:
yes...